axlearn/vision/attention.py

```diff
@@ -178,6 +181,8 @@ def forward(
             key:   an optional Tensor of shape [batch, source_length, source_dim].
             value: an optional Tensor of shape [batch, source_length, source_dim].
             attention_logit_biases:  See ``On attention logit biases`` in the file comments.
+            segment_ids: Not used. See `On segment_ids` in
```

Comment:
We may want to fix some of these docstrings too, since they are now used.

Reply:
Oops, see https://github.com/apple/axlearn/pull/719 for the fix.